selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/SystemAdminOrganizationViewsTest.java

```diff
@@ -24,151 +25,134 @@
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 import java.util.ArrayList;
-import org.eclipse.che.commons.lang.NameGenerator;
 import org.eclipse.che.multiuser.organization.shared.dto.OrganizationDto;
 import org.eclipse.che.selenium.core.client.TestOrganizationServiceClient;
-import org.eclipse.che.selenium.core.provider.TestDashboardUrlProvider;
-import org.eclipse.che.selenium.core.provider.TestIdeUrlProvider;
 import org.eclipse.che.selenium.core.user.AdminTestUser;
-import org.eclipse.che.selenium.core.user.TestUser;
-import org.eclipse.che.selenium.core.utils.WaitUtils;
 import org.eclipse.che.selenium.pageobject.dashboard.CheMultiuserAdminDashboard;
 import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationPage;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization views for admin of the organization.
+ * Test validates organization views for system admin.
  *
  * @author Ann Shumilova
  */
-public class AdminOfParentOrganizationTest {
-
-  private static final Logger LOG = LoggerFactory.getLogger(AdminOfParentOrganizationTest.class);
+public class SystemAdminOrganizationViewsTest {
+  private static final String PARENT_ORG_NAME = generate("organization", 5);
+  private static final String CHILD_ORG_NAME = generate("organization", 5);
```

Comment:
It's better to add **"child"** prefix to the name to simplify test result analysis.